pkgs/development/python-modules/pandas-stubs/default.nix

```
@@ -35,7 +35,7 @@
 
 buildPythonPackage rec {
   pname = "pandas-stubs";
-  version = "2.2.2.240603";
+  version = "2.2.2.240909";
   pyproject = true;
 
   disabled = pythonOlder "3.9";
```

```suggestion
  disabled = pythonOlder "3.10";
```
https://github.com/pandas-dev/pandas-stubs/blob/fa7e4446d226ee385412186bc3414a3499f42917/pyproject.toml#L31